### ISSUE TITLE
Netlify issue: missing node_modules packages. (resolved, but FYI)

### ISSUE THREAD

**Issue by jayroh** [mhanberg/jekyll-postcss#38]
In your README you mention that you must install postcss with npm or yarn.  I'd done this with `yarn add <package-name> -D`, which places things in the `devDependencies` hash key in `package.json`. This is fine. Typically running `bundle exec jekyll build` _locally_ will result in everything working just fine. Great!

Here's the hiccup: In many jekyll + postcss blog posts (eg: [here](https://stevenwestmoreland.com/2021/01/using-tailwind-css-with-jekyll.html)) there are instructions that instruct people to set the following ENV variables:

> When building for production, always set `JEKYLL_ENV=production` and `NODE_ENV=production` on the command line. When these variables are set to production, [unused Tailwind styles are purged](https://tailwindcss.com/docs/installation#building-for-production) and the output is processed with [cssnano](https://github.com/cssnano/cssnano).

This where the problem comes in. When deploying and building on Netlify `NODE_ENV=production` + `yarn add ______ -D` seems to result in an empty `./node_modules` directory.

When I moved everything from `devDependencies` to `dependencies` in `packages.json` then everything started working.

I don't know if this is a Netlify thing or not but, in the off-chance anyone shows up here with missing `node_modules` packages, this might be the answer.

***TLDR: Do not install your node dependencies via `yarn add .... -D` or `npm install <package-name> --save-dev`***
